components/two-factor-authentication/TwoFactorAuthenticationModal.tsx

```diff
@@ -90,6 +95,11 @@ export default function TwoFactorAuthenticationModal() {
     }
   }, [prompt]);
 
+  const closeModal = () => {
+    removeFromLocalStorage(LOCAL_STORAGE_KEYS.TWO_FACTOR_AUTH_TOKEN);
+    window.location.reload();
```

Comment:
Do we really need to reload the full page? Could we just close the modal and show a toast like `Two-Factor Authentication was aborted`?

Reply:
There must be other solutions; page reload should only be considered a last resort. Do you want to keep exploring this?